frontend/elements/src/entry/home/popup-logout.ts

```diff
@@ -0,0 +1,52 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+@customElement('jiggling-section')
```

Comment:
the filename is "popup-logout" but the element is "jiggling-section", this is confusing - please make them consistent

Reply:
Fixed